frontend/packages/api_client/src/models/OpenEdXAppServer.ts

```diff
@@ -84,14 +84,38 @@ export interface OpenEdXAppServer {
      * @memberof OpenEdXAppServer
      */
     edxPlatformCommit: string;
+    /**
+     * The repository to pull the default Ansible playbook from.
+     * @type {string}
+     * @memberof OpenEdXAppServer
+     */
+    ansibleAppserverRepoUrl?: string;
+    /**
+     * The path to the common appserver playbook to run on all appservers.
+     * @type {string}
+     * @memberof OpenEdXAppServer
+     */
+    ansibleAppserverPlaybook?: string;
+    /**
+     * The path to the requirements file for the common appserver playbook.
+     * @type {string}
+     * @memberof OpenEdXAppServer
+     */
+    ansibleAppserverRequirementsPath?: string;
+    /**
+     * The version of the Ansible playbook repository to checkout.
+     * @type {string}
+     * @memberof OpenEdXAppServer
+     */
+    ansibleAppserverVersion?: string;
```

Comment:
@0x29a, is there any reason to expose these fields via the API in the first place?

Reply:
@lgp171188, yep, we need to expose these fields to show their values in the Ocim interface (`Settings` tab):
![image](https://user-images.githubusercontent.com/18251194/111165389-a50c2000-85a7-11eb-8310-7b7410347c06.png)

Reply:
@0x29a. that doesn't use the new frontend APIs written in TS that this file is about, right? The settings tab in question uses the old APIs the clients for which are in written in Angular, no?

Reply:
@lgp171188 looks like both API versions use the [same serializer](https://github.com/open-craft/opencraft/blob/f99a37607c1d9ca45512091b72eb48fb1c170189/instance/serializers/openedx_instance.py#L93-L187). Should I do something about it?

Reply:
@demid, let's leave it as-is.